res/values/arrays.xml

```diff
@@ -2,74 +2,74 @@
+    <item>cyrl_ukr_jcuken</item>
+    <item>cyrl_rus_jcuken</item>
+    <item>deva_hin_inscript</item>
+    <item>deva_hin_lv</item>
+    <item>grek_ell_qwerty</item>
+    <item>hebr_heb_si_1452_1</item>
+    <item>hebr_heb_si_1452_2</item>
+    <item>kore_kor_ks_x_5002</item>
+    <item>latn_ces_qwertz</item>
+    <item>latn_deu_bone</item>
+    <item>latn_deu_neo2</item>
+    <item>latn_deu_qwertz</item>
+    <item>latn_eng_colemak</item>
+    <item>latn_eng_dvorak</item>
+    <item>latn_eng_qwerty</item>
+    <item>latn_eng_qwertz</item>
+    <item>latn_spa_qwerty</item>
+    <item>latn_fra_azerty</item>
+    <item>latn_lat_qwerty</item>
+    <item>latn_hun_qwerty</item>
+    <item>latn_hun_qwertz</item>
+    <item>latn_nor_qwerty</item>
+    <item>latn_pol_qwerty</item>
+    <item>latn_por_qwerty</item>
+    <item>latn_slk_qwertz</item>
+    <item>latn_swe_qwerty</item>
+    <item>latn_vie_qwerty</item>
+    <item>latn_tur_qwerty</item>
     <item>custom</item>
   </string-array>
   <string-array name="pref_layout_entries">
     <item>None</item>
-    <item>AZERTY</item>
-    <item>QWERTY</item>
-    <item>QWERTY (Brasileiro)</item>
-    <item>QWERTY (Español)</item>
-    <item>QWERTY (Greek)</item>
-    <item>QWERTY (Korean)</item>
-    <item>QWERTY (Latvian)</item>
-    <item>QWERTY (Norwegian)</item>
-    <item>QWERTY (Swedish)</item>
-    <item>QWERTY (Hungarian)</item>
-    <item>QWERTY (Türkçe)</item>
-    <item>QWERTY (Polski)</item>
-    <item>QWERTY (Vietnamese)</item>
-    <item>ЙЦУКЕН (Русский)</item>
-    <item>QWERTZ</item>
-    <item>QWERTZ (Czech)</item>
-    <item>QWERTZ (Deutsch)</item>
-    <item>QWERTZ (Hungarian)</item>
-    <item>QWERTZ (Slovak)</item>
-    <item>Bulgarian (Phonetic Traditional)</item>
-    <item>Dvorak</item>
-    <item>Colemak</item>
-    <item>Neo 2</item>
-    <item>Bone</item>
-    <item>ЙЦУКЕН (Українська)</item>
-    <item>বাংলা</item>
-    <item>देवनागरी (हिंदी)-1</item>
-    <item>देवनागरी (हिंदी)-2</item>
-    <item>Hebrew SI-1452-1</item>
-    <item>Hebrew SI-1452-2</item>
-    <item>Arabic PC</item>
-    <item>Arabic ALT</item>
+    <item>العربية: IBM</item>
+    <item>العربية: IBM [Mostafa Ayesh]</item>
+    <item>বাংলা: [Md Rasel Hossain]</item>
+    <item>български: ЯВЕРТЪ</item>
+    <item>русский: ЙЦУКЕН</item>
+    <item>українська: ЙЦУКЕН</item>
+    <item>हिंदी: InScript</item>
+    <item>हिदी: [Luke Videckis]</item>
+    <item>ελληνικά: QWERTY</item>
+    <item>עברית: ת״י 1452-1</item>
+    <item>עברית: ת״י 1452-2</item>
+    <item>한국어/조선말: 두벌식 KS X 5002</item>
+    <item>čeština: QWERTZ</item>
+    <item>Deutsch: Bone</item>
+    <item>Deutsch: Neo 2</item>
+    <item>Deutsch: QWERTZ</item>
+    <item>English: Colemak</item>
+    <item>English: Dvorak</item>
+    <item>English: QWERTY</item>
+    <item>English: QWERTZ</item>
+    <item>español: QWERTY</item>
+    <item>français: AZERTY</item>
+    <item>latviešu: QWERTY</item>
+    <item>magyar: QWERTY</item>
```

Comment:
should some of these be capitalized? (since we're standardizing)

Reply:
In the orthography of those languages, language names are not capitalised.

Reply:
Most of these languages capitalize the first word of a sentence. Also for consistency, I'd like to see every items capitalized.

Reply:
Hmm, there is a bit of different understanding around “standarisation”.
Probably because of Wikimedia Foundation's approach, in many places where autonyms of the language are used, other people capitalise them all, too.
If they want to make all languages of a writing system follow the same orthography rule, then it brings about another question: capitalise all or uncapitalise all. Outside the influence area of England, Germany, Netherlands, this rule of capitalising language names does not apply.
In my opinion, that Wikimedia Foundation's approach is a kind of anglicisation. So I chose not to force the capitalisation, or uncapitalisation, on other languages.

Reply:
Nope. The definition of a sentence is not like this. It is a noun phrase indeed.